src/ng/plugins/visibility/visibility.html

```diff
@@ -1 +1,16 @@
-<!--<span>{{$visibility.model.visibility()}}</span>-->
\ No newline at end of file
+<b>Visibillity: </b>
+<ul>
+	<li ng-repeat="(key, value) in $visibility.model.visibility() track by $index" ng-if="key !== 'resource'">
```

Comment:
Should be something like $visibility.items user who will overidde template shouldnt think about filtration